data/json/monsters/zed-winged.json

```
@@ -142,5 +143,49 @@
       "SWARMS",
       "FLIES"
     ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_spawn_raptor_shady",
+    "name": { "str_sp": "shady flesh-raptor" },
+    "copy-from": "mon_spawn_raptor",
+    "color": "light_gray",
+    "description": "An uncanny shadow envelops this creature.  You can make out the outline of three flapping wings connected to a jagged pointed object.",
+    "upgrades": {  },
+    "extend": { "flags": [ "NIGHT_INVISIBILITY" ] },
+    "vision_day": 5,
+    "vision_night": 40
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_spawn_raptor_unstable",
+    "name": { "str_sp": "unstable flesh-raptor" },
+    "copy-from": "mon_spawn_raptor",
+    "color": "red",
+    "description": "This small winged predator darts through the air on three thinly haired wings that look like stretched human hands.  Where the wings meet juts a jagged spike and a bloated fleshy belly.",
+    "special_attacks": [
+      [ "SUICIDE", 20 ],
+      { "id": "impale", "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_multiplier": 0.6 } ] }
+    ],
+    "death_function": [ "EXPLODE" ],
+    "upgrades": {  }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_spawn_raptor_electric",
+    "name": { "str_sp": "shocker flesh-raptor" },
```

```suggestion
    "name": { "str": "zaptor" },
```
THIS IS NOT UP FOR DEBATE! :O